l10n_th_tax_report/reports/tax_report_xlsx.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2019 Ecosoft Co., Ltd (https://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+import logging
+
+from odoo import models
+
```

Comment:
```suggestion
from odoo.addons.report_xlsx_helper.report.report_xlsx_format import (
    FORMATS,
    XLS_HEADERS,
)
```

add global file format